.dockerignore

```diff
@@ -0,0 +1,26 @@
+.git/
+.gitignore
+**/.gitkeep
+**/.DS_Store
+.dockerignore
+Dockerfile*
+docker-compose*.yml
+README.md
+certs/
+docs/
+notes/
+
+# pass env vars through docker-compose
+# or mount .env* in dev
+.env*
+env/
+dist/
+
+# named volumes
+node_modules/
+
+# cypress
```

Comment:
We use vitest for testing, maybe adapt the dockerignore to it? Or if the docker is for helping development (and not deployment), we don't need to ignore testing at all?